app/cdap/components/CreateAndEditRecipeForm/types.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { ISnackbar } from 'components/Snackbar';
+
+export interface ICreateAndEditRecipeFormProps {
```

Comment:
rename the CreateAndEditRecipeForm to RecipeForm - requested by Radhika
we need to make this name update across the changes in the PR
plz do it carefully, and test properly for UI/UX, validations, and unit tests
create and edit should be removed across the PR

Reply:
renamed to RecipeForm